ncm-spma/src/main/perl/spma/yum.pm

```diff
@@ -1138,11 +1138,11 @@ sub Configure
     $t->{run} = $t->{run} eq 'yes';
     $t->{userpkgs} = defined($t->{userpkgs}) && $t->{userpkgs} eq 'yes';
 
-    # When userpkgs are allowed, there is no control over what is in the reposdir
-    # If they are not allowed, and retry is allowed, use a non-standard location
-    # to avoid any repositories getting added during the retries (e.g. from rpms).
-    my $main_repos_dir = ($NoAction || $t->{userpkgs} || (!$t->{userpkgs_retry})) ?
-                             REPOS_DIR_DEFAULT : REPOS_DIR_QUATTOR;
+    # When userrepos are allowed, there is no control over what is in the reposdir
```

Comment:
But is that normal/expected? Although looking further down, isn't that what actually happens?